salt/search/init.sls

```diff
@@ -185,10 +195,10 @@ search-{{ process }}-service:
         - template: jinja
         - require:
             - aws-credentials-deploy-user
-            - search-ensure-index
```

Comment:
Doesn't sound needed

Reply:
on the follower node, the state `search-ensure-index` is not there because of its `if` evaluating to `False`, so requiring it should fail

Reply:
Is it needed on leader though? (Should be wrapped in an `if`?)

Reply:
`require_in` is the standard